src/storable.rs

```diff
@@ -421,6 +421,40 @@ where
     };
 }
 
+impl<T: Storable> Storable for Option<T> {
+    fn to_bytes(&self) -> Cow<[u8]> {
+        match self {
+            Some(t) => {
+                let mut bytes = t.to_bytes().into_owned();
+                bytes.insert(0, 1);
```

Comment:
Inserting the marker at the front can be inefficient for large values. Should we insert the marker at the end instead?